Csm-scrum-theory-guide.md

```
@@ -0,0 +1,417 @@
+| 重視するもの | 相対的に重視度が下がるもの |
+|---|---|
+| 個人と対話 | プロセスやツール |
+| 動くソフトウェア | 包括的なドキュメント |
+| 顧客との協調 | 契約交渉 |
+| 変化への対応 | 計画に従うこと |
+
+宣言文には「右記の事柄にも価値があることを認めながらも、私たちは左記の事柄をより重視する」という趣旨が明記されており、右側を全否定しているわけではない点に注意が必要です[9][10]。
+
+### 7.3 ScrumはアジャイルGの価値観・原則をどう体現しているか
+
+Scrum AllianceはCSM Learning Objectivesの根拠資料としてアジャイル宣言を明示的に挙げており[5][6]、ScrumはアジャイルGの価値観・原則を具体的な仕組みに落とし込んだフレームワークだと位置づけられます。代表的な対応関係を整理すると、次のようになります。
```

markdownlint-cli2 (0.23.2)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "6"

(MD052, reference-links-images)